### ISSUE TITLE
Wont work with DC integration

### ISSUE THREAD

**Issue by TysonTheEmber** [BONNePlayground/VaultHuntersDiscordIntegrator#2]
### Expected behavior

![image](https://github.com/BONNePlayground/VaultHuntersDiscordIntegrator/assets/101071342/917b8cd0-e3f3-46a2-8dad-950109209292)

### Observed/Actual behavior

![image](https://github.com/BONNePlayground/VaultHuntersDiscordIntegrator/assets/101071342/fdfd3f47-b93b-4b36-af8e-88f7569102cc)

### Steps/models to reproduce

install vh dc version 1.1.0(because 1.1.1 requires 1.19.4 for some reason)
install corresponding dc integration mod (2.5)

### VaultHunters version

latest (9.0.3)

**Comment by BONNe** [comment]
You need to upgrade Dcintegration for forge to version 2.6.4

They changes some parts of their API, cuz the update requirement and incompatibility with 2.5 version.

**Comment by BONNe** [comment]
Yeah... [Discord Integration for Forge](https://www.curseforge.com/minecraft/mc-mods/dcintegration) removed the 2.6.4 version from the 1.18.2 list apparently. 

It is still available via the CurseForge download repository, but not available to the public. Do not why it happened.

1.1.0 still works with 2.5.0 and VH update 9.

**Comment by TysonTheEmber** [comment]
thanks, i have been using it but it doesn't show data from chat shown items
also- https://modrinth.com/plugin/dcintegration/version/2.6.4-1.18.2

**Comment by TysonTheEmber** [comment]
the latest versions still dont show the chat items correctly
vh dc latest
dci https://modrinth.com/plugin/dcintegration/version/2.6.4-1.18.2

can you help?

**Comment by BONNe** [comment]
Does your Bot have permission to send embedded messages?
And does your chat supports Embedded messages?


Also, does `sendItemInfo` option in discord-integration.toml is enabled?

There is an option to test if it is this addon fault or just an issue with bot permissions:
- You need to share any enchanted vanilla tool in chat. If it still appears as [enchanted-tool] without description, then either `sendItemInfo` is disabled or your bot does not have permission (options) to send embed messages.

**Comment by TysonTheEmber** [comment]
there is no sendItemInfo option

**Comment by TysonTheEmber** [comment]
wait nvm and it is enabled

**Comment by TysonTheEmber** [comment]
I can not use commands from discord either, I have enabled perms and stuff.

**Comment by BONNe** [comment]
Have your bot have enabled these options:

![bot-settings](https://github.com/BONNePlayground/VaultHuntersDiscordIntegrator/assets/1344358/4f43c9e1-57d5-4f01-8c13-751a7c5d4add)

You can find them following this link:
https://discord.com/developers/applications/ 
Then click on `Bot` and there should be all these 3 options enabled.

**Comment by BONNe** [comment]
If it is enabled, then maybe it is related to the missing permissions to the channel.

You could add your bot to the channel permission set and give him embed links permission:
![embed-permission](https://github.com/BONNePlayground/VaultHuntersDiscordIntegrator/assets/1344358/8b578708-b12d-4aa4-9f44-fad1f46a776a)

**Comment by BONNe** [comment]
If none of these helps, then I would suggest asking support in https://github.com/ErdbeerbaerLP/DiscordIntegration-Forge/issues why would embed links not work with the bot. Maybe they know some more settings that need to be toggled.

**Comment by HuntorWorks** [comment]
This isnt working for me either... All the above suggestions are set to what you mentioned, yet whenever I send an item .. i get this 
[image](https://github.com/BONNePlayground/VaultHuntersDiscordIntegrator/assets/1016756/2990a0f3-e3dc-4ab5-9e8b-02b336e01726)
 and not the expected behaviour.  
 
 Using dcintergration-forge-3.0.3-1.18.2 and VaultHuntersDiscordIntegration-1.5.1-1.18.2 using VH 3.12.4

**Comment by BONNe** [comment]
Hmm, diamond_shovel is not a Vault Hunters item. So it would be triggered by DiscordIntegrations mod itself.
If removing `VaultHuntersDiscordIntegration-1.5.1-1.18.2` still results in an issue where `Diamond_shovel` with enchants are not displayed correctly, I would suggest to report a bug/support question for Discord Integrations mod author.

Personally, I use dcintergration-forge-3.0.3-1.18.2 and VaultHuntersDiscordIntegration-1.5.1-1.18.2 and it works on my server. So I assume it is something related to the server/bot settings or dcintegrations mod.
